README.md

```diff
@@ -18,6 +18,14 @@ npm i -S react-native-component-viewer
 
 - You may need to close and restart the React Native packager.
 
+# Dependencies
+
+This will require you to add the `Community AsyncStorage` to your project dependencies.
+
+```
+npm i -S @react-native-community/async-storage
+```
+
```

Comment:
Add here a link to the readme of `@react-native-community/async-storage` so people can follow the instructions to install that